web/profiles/joinup/config/install/og.og_role.rdf_entity-solution-member.yml

```diff
@@ -0,0 +1,11 @@
+langcode: en
+status: true
+dependencies: {  }
+id: rdf_entity-solution-member
+label: Member
+weight: 1
+group_type: rdf_entity
+group_bundle: solution
+group_id: null
+permissions: { }
+role_type: immutable
```

Comment:
More or less.! not an old Og version but I copied it from the collection roles. I updated both.